Look in the right place for ruby headers

This is an in-house patch, and has not been submitted upstream, since the
the changes are Solaris-specific.

--- graphviz-2.40.1/config/config_ruby.rb	2019-01-08 04:04:04.121480759 +0000
+++ graphviz-2.40.1/config/config_ruby.rb	2019-01-08 04:03:27.739180711 +0000
@@ -5,7 +5,7 @@ CONFIG = RbConfig::MAKEFILE_CONFIG
 
 case ARGV[0]
 when "archdir"
-    puts RbConfig::expand(CONFIG["archdir"])
+    puts "#{RbConfig::expand(CONFIG['rubyhdrdir'])} -I#{RbConfig::expand(CONFIG['rubyhdrdir'])}/#{RbConfig::expand(CONFIG['arch'])}"
 when "lib"
     puts RbConfig::expand(CONFIG["libdir"])
 when "vendorarchdir"
